ios/Podfile

```
@@ -47,18 +54,7 @@ target 'ObytesApp' do
       installer,
       config[:reactNativePath],
       :mac_catalyst_enabled => false,
-      :ccache_enabled => podfile_properties['apple.ccacheEnabled'] == 'true',
+      :ccache_enabled => ccache_enabled?(podfile_properties),
```

RuboCop (1.81.1)

[convention] Avoid comma after the last parameter of a method call.

(Style/TrailingCommaInArguments)